src/api/server/configuration.rst

```diff
@@ -16,316 +16,14 @@
 Configuration
 =============
 
-The CouchDB Server Configuration API provide an interface to query and update
-the various configuration values within a running CouchDB instance.
+The classical CouchDB Server Configuration API located at ``/_config`` will
+return a 404 in CouchDB 2.0 and got replaced by :ref:`HTTP API <api/node>`.
 
-``/_config``
```

Comment:
API documentation is not about our position what people should use and what they shouldn't. It's about the product and what product is able to provide. Discouraging things is about best practices which could be implemented with a warning note.

Also, you forgot the case when node gets out of the cluster - then you'll not able to use cluster API to get any information from it because:

```
HTTP/1.1 500 Internal Server Error
Cache-Control: must-revalidate
Content-Length: 54
Content-Type: text/plain; charset=utf-8
Date: Sun, 12 Jul 2015 15:31:56 GMT
Server: CouchDB/bcb7c1a (Erlang OTP/18)
X-Couch-Request-ID: 3ca2e2d26b
X-CouchDB-Body-Time: 0

{"error":"nodedown","reason":"progress not possible"}
``` 

And them this happens, you need to find out how to get your data back. If we're going to hide all information about local interface and even remote it, we doom those people that will have to deal with such cases. I don't see any reason why we should make life complicated for our users while we can avoid it.